lib/cucumber/http/http_steps.rb

```diff
@@ -5,6 +5,36 @@
   hdrs.rows_hash.each { |name, value| add_header(name, value) }
 end
 
+Given /^I send "(.*?)"$/ do |content_type|
+  steps %Q{
+    Given I set headers:
+      | Content-Type | application/#{content_type} |
+  }
+end
+
+Given /^I accept "(.*?)"$/ do |accept_type|
+  steps %Q{
+    Given I set headers:
+      | Accept       | application/#{accept_type} |
+  }
+end
+
+Given /^I send and accept "(.*?)"$/ do |type|
+  steps %Q{
+    Given I set headers:
+      | Content-Type | application/#{type} |
+      | Accept       | application/#{type} |
```

Comment:
Won't this set the headers to `application/application/json`? (Following the `And I send and accept "application/json"` example in the README)

Reply:
Yes, you are right! Too much copy/paste :)